netci.groovy

```diff
@@ -4,6 +4,8 @@
 import jobs.generation.Utilities;
 def project = GithubProject
 
+buildTimeLimit = 120
```

Comment:
Both of your statements begin with "Without def" and describe different behavior.  Pretty sure you meant to use "With def" on one of them.  .  :smile: 

> I know hardly anything about groovy, so let me know if I'm wrong about this.

Same. Double worse is the only way I know to test / run groovy is to submit this script to Jenkins and watch it fail / succeed.